cypress/e2e/company_interest_spec.js

```diff
@@ -23,8 +23,6 @@ const createCompanyInterest = () => {
   field('events[0].checked').check();
   field('otherOffers[0].checked').check();
 
-  field('comment').type('random comment');
```

Comment:
Why is this removed? Wasn't this just renamed to `aboutTheCompany`?

Reply:
though we changed it, and regreted it so changed it back ;)